include/identity.php

```diff
@@ -656,8 +656,10 @@ function advanced_profile(App $a) {
 			$profile['birthday'] = array( t('Birthday:'), $val);
 
 		}
-
-		if ($age = age($a->profile['dob'],$a->profile['timezone'],'')) {
+		if ($a->profile['dob']
```

Comment:
Just change it into an ```!empty``` please. Although this case should never happen that this array element doesn't exist, it is more clean.